client/sources/common/interpreter.py

```diff
@@ -289,7 +289,7 @@ def _interpret_lines(self, lines, compare_all=False):
                         return False
                     current = []
                 # avoid printing reload import sequence
-                if line and (compare_all or (self.fpp and i > 4)):
+                if line and (self.fpp and i > 4):
```

Comment:
so does this mean that lines will only be printed if we're in an fpp setting?

Reply:
Thought about this a bit more and the whole purpose of this was to not print the reload sequence, which only occurs when `self.fpp` is `True`. Originally the code was 
```
if line:
   print(line)
   ```
and I don't want to change this behavior. Will update in next commit

Reply:
okay sounds good, so something like this?

```py
if line and (not self.fpp or i > 4):
    print(line)
```

Reply:
Update: Fixed the logic of this so that it doesn't print when the function is called on the import sequence